frontend/ui/widget/menu.lua

```diff
@@ -657,6 +607,9 @@ local Menu = FocusManager:new{
     -- if you want to embed the menu widget into another widget, set
     -- this to false
     is_popout = true,
+    -- set this to true to add extra (left) button to the title bar
+    has_extra_button = false,
```

Comment:
I somewhat overlooked this earlier, but #8569 made me realize I'm not in love with this name. Though admittedly, I'm not sure `show_extra_button` or something is necessarily any better.

Reply:
The name is consistent with
https://github.com/koreader/koreader/blob/03253ef2d1c19b0e45e2c4636c0004a3ee3ef100/frontend/ui/widget/menu.lua#L661

Reply:
My thoughts are identical, but carry on. ;-)